app/src/main/java/org/p2p/wallet/utils/Base58String.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.wallet.utils
+
+import org.p2p.solanaj.utils.crypto.Base58Utils
+
+class Base58String(val value: String) {
```

Comment:
class that i hope will help us to move from primitive obsession and use force hidden in the types!